### PR TITLE
Improve DApp Performance

### PR DESCRIPTION
**Pull request by Seroxdesign**
Closes: #397

### CODE REVIEW

**Comment by Seroxdesign** [comment]
Testing for now hold off on review please

**Review by pi0neerpat — CHANGES_REQUESTED** [review]
- delete logo192, assets/splash folder, od-land. od-vault. staking

**Review by pi0neerpat — APPROVED** [review]
Normally we re-request, rather than dismiss reviews

**Pull request merged**
